mlir/test/mlir-miopen-driver/populate_host_convert.mlir

```diff
@@ -40,23 +40,3 @@
 // F16-NEXT:   }
 // F16-NEXT: }
 // F16-NEXT: return
-
-// BF16: func @convert_result([[SOURCE:%[a-zA-Z_0-9]+]]: memref<[[N:[0-9]+]]x[[K:[0-9]+]]x[[HO:[0-9]+]]x[[WO:[0-9]+]]x[[TYPE:[a-zA-Z0-9]+]]>, [[DEST:%[a-zA-Z_0-9]+]]: memref<[[N]]x[[K]]x[[HO]]x[[WO]]x[[PRINT_TYPE:[a-zA-Z0-9]+]]>)
```

Comment:
add test to check bf16-specific conversion routine is invoked.

Reply:
we use extern function to convert , the test is in populate_host_print.mlir

Reply:
test added